fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/Traversal.java

```diff
@@ -92,7 +92,11 @@ public Set<Reference> getLeafReferences() {
      */
     public Set<Reference> getRefsContaining(final RelationalExpression expression) {
         final var result = containedInMultiMap.get(expression);
-        Debugger.sanityCheck(() -> Verify.verify(result.stream().allMatch(ref -> ref.getMembers().stream().anyMatch(e -> e == expression))));
+        Debugger.sanityCheck(() ->
+                Verify.verify(result.stream()
+                        .allMatch(ref -> ref.getAllMemberExpressions()
+                                .stream()
+                                .anyMatch(e -> e == expression))));
```

Comment:
Coverage analysis suggests that this method is never called

Reply:
This code lights up because of the `allMemberExpressions()` change + a bunch of indentation. Please open an issue for the coverage problems in general.